test_vm/Vagrantfile

```diff
@@ -34,7 +46,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         "password" => {
           "postgres" => "postgres"
         },
-        "version" => "9.3",
+        "version" => "9.4",
```

Comment:
I've got one the Postgres upgrade related changes from the `test-vm-update-postgresql-to-9.4` branch.